src/wallet/rpcwallet.cpp

```diff
@@ -2174,7 +2174,7 @@ UniValue settxfee(const UniValue& params, bool fHelp)
     if (fHelp || params.size() < 1 || params.size() > 1)
         throw runtime_error(
             "settxfee amount\n"
-            "\nSet the transaction fee per kB.\n"
+            "\nSet the transaction fee per kB. (Overwrites the paytxfee parameter)\n"
```

Comment:
Nit: Either 
- "Set the transaction fee per kB (overwrites the paytxfee parameter)." or
- "Set the transaction fee per kB. (Overwrites the paytxfee parameter.)"

(http://english.stackexchange.com/questions/6632/where-does-the-period-go-when-using-parentheses)